src/analysis/convex/extreme.lean

```diff
@@ -35,8 +35,6 @@ See chapter 8 of [Barry Simon, *Convexity*][simon2011]
 
 ## TODO
 
-Define intrinsic frontier and prove lemmas related to extreme sets and points.
```

Comment:
The first part of this is still TODO, right?

Reply:
Argh! Copy-paste accident.